src/inference/src/dev/core_impl.hpp

```diff
@@ -183,6 +183,12 @@ class CoreImpl : public ov::ICore, public std::enable_shared_from_this<ov::ICore
     }
     void add_extensions_unsafe(const std::vector<ov::Extension::Ptr>& extensions) const;
 
+    Plugin get_plugin(const std::string& pluginName, bool on_create_filter_config) const;
+
+    AnyMap get_supported_property(const Plugin& plugin, const AnyMap& config, bool keep_core, bool rw_only) const;
+
+    AnyMap get_hw_plugin_properties_or_forward(const Plugin& plugin, const AnyMap& config, bool rw_only = false) const;
```

Comment:
Why isn't `rw_only` defaulted in both declarartions?

Reply:
Removed from `get_hw_plugin_properties_or_forward` this parameter as it is not required now.